src/woo_publications/accounts/models.py

```diff
@@ -69,3 +71,29 @@ def get_full_name(self):
     def get_short_name(self):
         "Returns the short name for the user."
         return self.first_name
+
+
+class OrganisationMember(models.Model):
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_(
+            "The system identifier that uniquely identifies the user performing "
+            "the action."
+        ),
+        max_length=255,
+        unique=True,
+    )
+    naam = models.CharField(
+        _("naam"),
+        help_text=_("The display name of the user."),
+        max_length=255,
+    )
+
+    objects: ClassVar[OrganisationMemberManager] = OrganisationMemberManager()  # pyright: ignore [reportIncompatibleVariableOverride]
```

Comment:
![image](https://github.com/user-attachments/assets/ed621336-2e04-430c-9bc7-3d0018fb430e)

I couldn't get rid of this type error so I ignored it.